index.html

```diff
@@ -15,255 +15,261 @@
-          <p>Get my resume</p>
-        </a>
-        <ul class="social">
-          <li class="socialicon">
-            <a href="https://stackoverflow.com/users/17562225/mukhammadrizooff"
-              ><img src="icons/ic_stackoverflow.png" alt="stackoverflow"
-            /></a>
-          </li>
-          <li class="socialicon">
-            <a href="https://github.com/mukhammadrizooff"
-              ><img src="icons/ic_github.png" alt="Github"
-            /></a>
-          </li>
-          <li class="socialicon">
-            <a href="#"><img src="icons/ic_twitter.png" alt="twitter" /></a>
+      <div class="container-about">
+        <h2 class="aboutheader">About</h2>
+        <p class="introabout">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut,
+        </p>
+        <button type="button" class="btn2">Get my CV</button>
+      </div>
+      <div class="container-about2">
+        <ul class="grid-container">
+          <li class="aboutcard">
+            <ul class="grid-item1">
+              <li class="about-title">
+                <h3>Languages</h3>
+              </li>
+              <li>JavaScript (ES5/ES6)</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
           </li>
-          <li class="socialicon">
-            <a href="https://www.linkedin.com/in/mukhammadrizooff"
-              ><img src="icons/ic_linkedin.png" alt="linkedin"
-            /></a>
+          <li class="aboutcard">
+            <ul class="grid-item2">
+              <li class="about-title">
+                <h3>Frameworks</h3>
+              </li>
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capibara</li>
+              <li>Selenium</li>
+            </ul>
           </li>
-          <li class="socialicon">
-            <a href="https://medium.com/@mukhammadrizooff"
-              ><img src="icons/ic_medium.png" alt="Medium"
-            /></a>
+          <li class="aboutcard">
+            <ul class="grid-item3">
+              <li class="about-title">
+                <h3>Skills</h3>
+              </li>
+              <li>Codekit</li>
+              <li>Codepen</li>
+              <li>Github</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+            </ul>
           </li>
-          <li class="socialicon">
-            <a href="#"><img src="icons/ic_angellist.png" alt="Angelist" /></a>
+          <li class="aboutcard">
+            <ul class="grid-item4">
+              <li><img src="images/dots.png" width="170" alt="dots" /></li>
+            </ul>
           </li>
         </ul>
-      </footer>
+      </div>
     </section>
+    <form action="https://formspree.io/f/xjvlbplk" method="post" id="form">
+      <fieldset class="contact">
+        <h2 class="contactheader">Get Started</h2>
```

Comment:
- [ ] You have done a good job with the contact section but I think it could be a good idea to add some spacing above this title to match the Figma design :wink:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144728526-8cfea866-f5a5-4da3-9f52-706dee511b74.png)


  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144728537-05a2c412-3f65-4ad7-95f0-4325170c44b4.png)

Reply:
This change was not implemented. Kindly implement it. 🙏

Reply:
Ok